.github/workflows/app-lint.yaml

```diff
@@ -0,0 +1,21 @@
+name: 'Frontend Pull Request Lint'
+on:
+  pull_request:
+    branches: [ 'main' ]
+    paths:
+      - "src/frontend/**"
+  workflow_dispatch:
+
+jobs: 
+  API-unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Navigate to Backend, run npm tests
```

Comment:
typo: Navigate to "Frontend"

Reply:
whoopsie, copy pasta. thanks for catching that.